taccsite_cms/static/site_cms/css/src/site.header.css

```diff
@@ -12,9 +12,6 @@
            variables are unfortunately overwritten 1x in CMS, 2x in Portal */
 /* TODO: Find a way to be modular, but not redundant; perhaps `env()` vars */
 @import url("_imports/settings/font.css");
-/* NOTE: Portal does not need this, because it has independent copy of these,
-         but the User Guides do not. This can be removed in Core-CMS PR #192 */
-@import url("_imports/settings/color.css");
```

Comment:
**Why?** Because the comment told me to—"This can be removed in Core-CMS PR #192"—and I trust me.